src/app/(pages)/experience/_components/ExperienceDetailContent.tsx

```
@@ -266,6 +471,161 @@ export function ExperienceDetailContent({
+      displayValue: string;
+      name: BasicDetailKey;
+    };
+
+function getDetailInfoItems(
+  type: ExperienceItem['type'],
+  basicDetail: ExperienceItem['basicDetail'],
+  periodLabel: string,
+): EditableDetailInfoItem[] {
+  switch (type) {
+    case 'activity':
+      const teamNum = basicDetail.teamNum ?? '';
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)